### PR TITLE
Creating Landing page, Privacy Notice Page and Tearms and Conditions Page

### PR DESCRIPTION
**Pull request by marija-cvetkovic-ey**
- Creating Landing page with carousel
- Getting data from NASA APIs -https://api.nasa.gov/ with axios
- Creating Privacy Notice page
- Creating Tearms and Conditions page

### CODE REVIEW

File: src/components/App/Footer.js [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
+import './style.scss';
 const Footer = () => {
     return (
-        <div>
-            <h1>This is Footer</h1>
-            <Link to="/terms-and-conditions">Terms and Condition</Link>
-            <Link to="/privacy-note">Privacy Notice</Link>
-        </div>
+        <footer className="footer">
+            <Link className="footer-links" to="/terms-and-conditions">Terms and Condition</Link>
```
Comment (MilanovicM) [inline src/components/App/Footer.js:7]: The text that will be visible to the user should be placed in some separate file for constants. This way, if we later want to change the text, we can do it in one place. Or we can for example add translation library, and get the values from one file.

File: src/components/App/Header.js [hunk 2]
```diff
@@ -1,11 +1,12 @@
 import React from 'react'
 import { Link } from 'react-router-dom';
+import logo from '../../assets/logo.png';
+import "./style.scss";
 const Header = () => {
     return (
```
Comment (MilanovicM) [inline src/components/App/Header.js:6]: This is a good place to use implicit return in component :)

File: src/components/Carousel/Carousel.js [hunk 3]
```diff
@@ -1,11 +1,97 @@
...
+    const [currentPage, setCurrentPage] = useState(0);
+    const width = useWindowSize();
+    const [cardPerPage, setCardPerPage] = useState(1);
+    const [paginationArray, setPaginationArray] = useState([]);
+
+    const createPaginationArray = useCallback(
+        (cardPerPage, length) => {
+            let array = [];
+            for (let i = 0; i < Math.ceil(length / cardPerPage); i++) {
+                array.push(i);
```
Comment (MilanovicM) [inline src/components/Carousel/Carousel.js:17]: Is there some reason why are we not using Array.prototype method map? Just asking :)

File: src/components/Home/Home.js [hunk 4]
```diff
@@ -1,10 +1,21 @@
 import React from 'react'
 import Carousel from '../Carousel/Carousel';
+import Button from '../Button/Button';
+import './style.scss';
+import GetPictures from '../../services/GetPictures';
+import Error from '../App/Error';
+import Loader from '../App/Loader';
 const Home = () => {
+    const count = 16;
+    const { data, isPanding, error } = GetPictures(`https://api.nasa.gov/planetary/apod?api_key=h3gXgkZ5F9vwa8A4Q1ZLyZlSIwcw1xkWdUr55Qev&count=${count}`);
```
Comment (MilanovicM) [inline src/components/Home/Home.js:10]: Maybe this string for NASA API endpoint could be moved to separate constants file?

File: src/components/Home/Home.js [hunk 5]
```diff
@@ -1,10 +1,21 @@
...
-        <div className="Home">
-            <h1>This is home page</h1>
-            <Carousel />
+        <div className="home">
+            {error && <Error error={error} />}
+            {isPanding && <Loader />}
+            {data && <Carousel data={data} />}
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+                incididunt ut labore et dolore magna aliqua.</p>
+            <Button text={"Start Application Process"} />
```
Comment (MilanovicM) [inline src/components/Home/Home.js:18]: I think we don't need to use curly braces to pass the string here.

File: src/components/Carousel/Carousel.js [hunk 6]
```diff
@@ -1,11 +1,97 @@
...
+                    )
+                }
+                )}
+                <div className="arrow"><button onClick={() => {
+                    handleNextChangePage(currentPage)
+                }}>&raquo;</button></div>
+            </div>
+            <div className="pagination">
+                {paginationArray != null && paginationArray.map((index) => {
+                    return (
```
Comment (MilanovicM) [inline src/components/Carousel/Carousel.js:89]: Index here represent the element form array, correct? It's a bit confusing to use that name for the element :)

**Review by MilanovicM — APPROVED** [review]
##### Excellent work Marija!
Please see my comments below:
 - General findings:

- Axios missing in package json, so the application couldn't be started.
- PR has clear title and description
- Maybe Router could be moved to separate file
- Good thing - NotFound root exists
- No default props or  prop types
- The UI is exceptionally nice - kudos!
- service GetPicture could be renamed and used as a service for making HTTP GET requests
Great work Marija :)

File: src/components/Carousel/Carousel.js (hunk 3 above)
Comment (marija-cvetkovic-ey) [reply src/components/Carousel/Carousel.js:17]: Because I am used to using for loops :) But I will try with map method.

**Pull request merged**